### PR TITLE
Added AWS native GuardDuty integration documentation

### PR DESCRIPTION
**Pull request by EduLeon12**
## Description

GuardDuty implementation now supports native integration to S3 buckets, replacing the current one which requires AWS Kinesis/Firehose/CloudWatch integration to send related logs.

Documentation has been updated to include a step by step guide on how to enable it.

## Checks
- [x] Compiles without warnings.
- [x] Uses present tense, active voice, and semi-formal registry.
- [x] Uses short, simple sentences.
- [x] Uses **bold** for user interface elements, _italics_ for key terms or emphasis, and `code` font for Bash commands, file names, REST paths, and code.
- [x] Uses three spaces indentation.
- [x] Adds or updates meta descriptions accordingly.
- [x] Updates the `redirects.js` script if necessary (check [this guide](https://github.com/wazuh/wazuh-documentation/blob/master/NEW_RELEASE.md)).

### CODE REVIEW

**Review by nico-stefani — CHANGES_REQUESTED** [review]
Please add an entry to https://documentation.wazuh.com/current/amazon/services/supported-services/index.html. Because the path for native logs is different.

File: source/amazon/services/supported-services/index.rst [hunk 1]
```diff
@@ -33,7 +33,9 @@ The next table contains the most relevant information about configuring each ser
 +--------------+----------------------------------------------------------+-----------------------+----------------+------------------------------------------------------------------------------------------------------------------+
 | Amazon       | :ref:`Trusted Advisor <amazon_trusted_advisor>`          | bucket                | custom         | <bucket_name>/<prefix>/<year>/<month>/<day>                                                                      |
 +--------------+----------------------------------------------------------+-----------------------+----------------+------------------------------------------------------------------------------------------------------------------+
-| Amazon       | :ref:`GuardDuty <amazon_guardduty>`                      | bucket                | guardduty      | <bucket_name>/<prefix>/<year>/<month>/<day>/<hh>                                                                 |
+| Amazon       | :ref:`GuardDuty Native <amazon_guardduty_native>`        | bucket                | guardduty      | <bucket_name>/<prefix>/<year>/<month>/<day>/<hh>                                                                 |
```
Comment (nico-stefani) [inline source/amazon/services/supported-services/index.rst:36]:
```suggestion
| Amazon       | :ref:`GuardDuty Native <amazon_guardduty_native>`        | bucket                | guardduty      | <bucket_name>/<prefix>/AWSLogs/<suffix>/<account_id>/GuardDuty/<region>/<year>/<month>/<day>                                                                 |
```

**Review by nico-stefani — CHANGES_REQUESTED** [review]

**Review by nico-stefani — APPROVED** [review]
LGTM

**Review by davidjiglesias — APPROVED** [review]

**Comment by EduLeon12** [comment]
File updated to comply with new guidelines and requested changes

**Review by s-ocando — APPROVED** [review]

**Pull request merged**